csp_billing_adapter/memory_csp_config.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2023 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+
+import csp_billing_adapter
+
+from csp_billing_adapter.config import Config
+
+memory_csp_config = {}
+
+log = logging.getLogger('CSPBillingAdapter')
+
+
+@csp_billing_adapter.hookimpl(trylast=True)
```

Comment:
Defaults in the hooks, I thought we didn't want those? Did I miss something?

Reply:
That is not a default parameter for the hook call, it is the argument to the hookimpl decorator, telling it to add the hook at the end of the list of hooks rather than at the start of the list as is the default. See #34 for details and an example of the problem I was seeing.